src/backup_camera.h

```diff
@@ -25,13 +25,17 @@ extern "C" {
 #include "WiringPiButtons.hpp"
 #endif
 
-//#include "MusicBar.h"
-//#include "SongPlayer.h"
+
 #include "graphics_handler.h"
 #include "videoStream.hpp"
 
-
-
+namespace cameraVariables
```

Comment:
Give the two namespaces different names. 

QUIT_EVENT_FLAG
ENTER_FULLSCREEN_EVENT_FLAG
EXIT_FULLSCREEN_EVENT_FLAG

Can probably be in an anonymous namespace in the cpp file.